FetchMigration/Dockerfile

```diff
@@ -14,5 +14,5 @@ COPY python/*.py .
 # update PATH
 ENV PATH=/root/.local:$PATH
 
-# make sure you include the -u flag to have our stdout logged
-ENTRYPOINT python -u ./main.py -r $ICT_CODE_PATH/input.yaml $DATA_PREPPER_PATH/pipelines/pipelines.yaml; $DATA_PREPPER_PATH/bin/data-prepper
+# Include the -u flag to have stdout logged
+ENTRYPOINT python -u ./orchestrator.py $DATA_PREPPER_PATH $ICT_CODE_PATH/input.yaml https://localhost:4900
```

Comment:
Since this will be situated within something else (which may also be situated within something else, because you never know) - you might want to disambiguate "orchestrator" with "fetch" (e.g. fetchOrchestrator.py)

What does ICT stand for?

Reply:
Renamed `orchestrator.py` to `fetch_orchestrator.py`

ICT = Index Configuration Tool. This is left over from when the tool would only perform index configuration rather than the full fetch migration workflow. Changed references to `ICT` to `FM` instead.